meta-cli/Cargo.toml

```diff
@@ -37,7 +41,9 @@ tokio = { version = "1.33.0", features = ["full"] }
 futures = "0.3.28"
 log = "0.4.20"
 env_logger = "0.10.0"
-clap = { version = "4.4.6", features = ["derive", "env"] }
+clap = { version = "=4.3.3", features = ["derive", "env"] }
```

Comment:
Why that specific version?

Reply:
The `deno` crate pulls in that version and it triggered that edge case where cargo refuses to compile because "couldn't choose between two versions". It happily compiles duplicate versions of the same crate elsewhere, I ought to look into what causes that. But yeah, I opted to downgrade as we did not seem to be relying on anything from 4.4 and duplicates means bloat.

Reply:
Could we add a comment?